test/evolve.test.mjs

```diff
@@ -44,8 +44,8 @@ describe('aos Evolve', async () => {
       },
       evolveResult.Memory,
     );
-    assert(evalResult.Output.data.output);
-    assert(evalResult.Output.data.output.includes('info'));
+    assert(evalResult.Output.data);
```

Comment:
Wasn't this like an AOS 1.x -> 2.x change? Interesting that it only came up now?

Reply:
Sign of the age of our test fixture - but since we were deploying with that, this was an accurate test. Now that we have updated its the time to change.